pvanalytics/quality/weather.py

```diff
@@ -1 +1,75 @@
 """Quality control functions for weather data."""
+from pvanalytics.quality import util
+
+
+def temperature_limits(air_temperature, limits=(-35.0, 50.0)):
+    """Identify extreme temperatures.
+
+    Parameters
+    ----------
+    air_temperature : Series
+        Air temperature in Celsius.
```

Comment:
For discussion: do we want to use RTD auto-format for units?  It would be 
```suggestion
        Air temperature [C].
```

Reply:
This seems like a good idea. Can you point me towards some documentation for this feature? In other parts of the code I have been using something like `` :math:`W/m^2` `` to format units like 'W/m^2', but if possible I would much prefer [W/m^2]

Reply:
So, looking at it again, I don't think it's actually formatting anything, just putting text braces around the unit. pvlib tries to stick with SI unit symbols. One [example ](https://pvlib-python.readthedocs.io/en/stable/generated/pvlib.temperature.sapm_cell.html) docstring using the braces. I think you could [`:math:`W/m^2`] if we want to format with superscripts.